components/Layout/Navigation.js

```diff
@@ -13,19 +13,19 @@ import Link from '../Link';
 import s from './Navigation.css';
 
 class Navigation extends React.Component {
-  //
-  // componentDidMount() {
-  //   window.componentHandler.upgradeElement(this.root);
-  // }
-  //
-  // componentWillUnmount() {
-  //   window.componentHandler.downgradeElements(this.root);
-  // }
+   //
+   //componentDidMount() {
+   //  window.componentHandler.upgradeElement(this.root);
+   //}
+   //
+   //componentWillUnmount() {
+   //  window.componentHandler.downgradeElements(this.root);
+   //}
 
   render() {
     return (
       <div className={s.menu}>
-        <div className="mobile-menu">
+        <div className={s.mobile-menu}>
```

Comment:
de ce mobile-menu e ca clasa din css, unde ai in Navigation.css clasa mobile-menu ?